docs/setup.md

```diff
@@ -0,0 +1,14 @@
+# ORCID auth instructions
+
+#### Steps to obtain ORCID
```

Comment:
I meant steps to create an app with ORCID to get the env keys like `ORCID_AUTH_CLIENT_ID` & `ORCID_AUTH_CLIENT_SECRET`. Did you record that anywhere?